lib/blocs/drive_detail/drive_detail_cubit.dart

```diff
@@ -395,10 +411,11 @@ class DriveDetailCubit extends Cubit<DriveDetailState> {
   }
 
   void refreshDriveDataTable() {
-    _refreshSelectedItem = true;
+    // _refreshSelectedItem = true;
 
     if (state is DriveDetailLoadSuccess) {
-      emit((state as DriveDetailLoadSuccess).copyWith());
+      final state = this.state as DriveDetailLoadSuccess;
+      emit(state.copyWith());
```

Comment:
```suggestion
      emit(state as DriveDetailLoadSuccess);
```

We don't need to reassign state and also we don't need an empty copyWith